Kiosk/App/Views/RegisterFlowView.swift

```diff
@@ -111,3 +121,11 @@ class RegisterFlowView: ORStackView {
         }
     }
 }
+
+extension Array {
```

Comment:
Out of curiosity, in Swift, how do extensions work? Does it extend `Array` in all scopes, or just where the extension is in scope?